projects/packages/videopress/src/client/admin/components/video-details-card/types.ts

```diff
@@ -12,24 +12,24 @@ export type VideoThumbnailProps = {
 	/**
 	 * Whether is possible to edit the thumbnail
 	 */
-	editable: true;
+	editable?: true;
```

Comment:
Shouldn't this be `boolean`?

Reply:
https://github.com/Automattic/jetpack/pull/25981/commits/68250662f6f202da1f75dd19db6cacdc87d6359f